custom-script/utils.py

```diff
@@ -166,6 +198,18 @@ def plot_comparison(rgb_array, sh_prediction, model_prediction, threshold=0.5):
     ax[1][2].set_title("[d] water prediction with model, locally", fontsize=14)
 
     ax[1][3].imshow(sh_thr - model_thr, vmin=-1, vmax=1, cmap="RdBu")
-    ax[1][3].set_title("differences between [d] and [d]", fontsize=14)
+    ax[1][3].set_title("differences between [c] and [d]", fontsize=14)
 
     plt.tight_layout(pad=0.05)
+
+
+def plot_visualize(patch, factor=3.15):
```

Comment:
I understand that all plotting methods have `plot_` in them, but `plot_visualize` doesn't make much sense. `plot_eopatch` would be better, imo.

Reply:
Moved into note book (done).